lib/omnisharp-atom/features/code-action.ts

```diff
@@ -0,0 +1,30 @@
+import Omni = require('../../omni-sharp-server/omni')
+import OmniSharpAtom = require('../omnisharp-atom')
+import SpacePen = require('atom-space-pen-views');
+import CodeActionsView = require('../views/code-actions-view');
+
+class CodeCheck {
+
+    view;
+
+    constructor(private atomSharper: typeof OmniSharpAtom) {
+        this.atomSharper = atomSharper;
+    }
+    public activate() {
+        atom.commands.add('atom-workspace', "omnisharp-atom:get-code-actions", () => {
```

Comment:
we just added OmniSharpAtom.addCommand, so you can do `OmniSharpAtom.addCommand("omnisharp-atom:get-code-actions", () => {`

This takes care of making sure that the command will only run for a c# file.